flow/connectors/postgres/postgres.go

```diff
@@ -284,7 +283,7 @@ func (c *PostgresConnector) SyncRecords(req *model.SyncRecordsRequest) (*model.S
 	for record := range req.Records.GetRecords() {
 		switch typedRecord := record.(type) {
 		case *model.InsertRecord:
-			itemsJSON, err := typedRecord.Items.ToJSON()
+			itemsJSON, err := typedRecord.Items.ToJSON(false)
```

Comment:
why not json encode hstore here?

Reply:
We want to write hstore as hstore to postgres

Reply:
refactored to separate method